injected/entry-points/integration.js

```diff
@@ -26,7 +26,14 @@ function generateConfig() {
             domain: topLevelUrl.hostname,
             isBroken: false,
             allowlisted: false,
-            enabledFeatures: ['fingerprintingCanvas', 'fingerprintingScreenSize', 'navigatorInterface', 'cookie'],
+            enabledFeatures: [
+                'fingerprintingCanvas',
+                'fingerprintingScreenSize',
+                'navigatorInterface',
+                'cookie',
+                'webCompat',
+                'apiManipulation',
```

Comment:
this change is needed to keep the tests passing. In production the extension calls 'load' on every feature.